contracts/plugins/assets/convex/CvxStableRTokenMetapoolCollateral.sol

```diff
@@ -82,4 +82,9 @@ contract CvxStableRTokenMetapoolCollateral is CvxStableMetapoolCollateral {
         rTokenOracle.price(IRToken(address(pairedToken)), false); // refresh price in oracle
         super.refresh();
     }
+
+    // Override CvxStableMetapoolCollateral, since paired RToken cannot default
+    function _anyDepeggedOutsidePool() internal view virtual override returns (bool) {
+        return false;
```

Comment:
why not check the RTokenPriceOracle here?

Reply:
I don't think we check for depegged state for RTokens, but maybe we should.

Reply:
Added